files/default/neutron-ha-tool.py

```diff
@@ -579,6 +579,16 @@ def list_agents(qclient, agent_type=None):
     return resp['agents']
 
 
+def list_alive_agents(qclient, agent_type=None):
+    resp = qclient.list_agents()
+    LOG.debug("list_agents: %s", resp)
+    if agent_type:
+        return [agent for agent in resp['agents']
+                if agent['agent_type'] == agent_type and
+                agent['alive'] is True]
```

Comment:
@aspiers I'm talking about this line: https://github.com/SUSE-Cloud/cookbook-openstack-network/pull/2/files/18079d9811e08cc938efa055bdb5b797c1b0e11d#diff-937ed0b9cba9efd2521c8fd1c9e8416fR585

This line doesn't filter anything; it's just checking if the parameter is `None` or not.